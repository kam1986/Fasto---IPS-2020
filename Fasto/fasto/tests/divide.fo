
fun int main() =
    let t1 = write(4/2) in
    let t2 = write(~4/ ~2) in
    let t3 = write(2/0) in
    1