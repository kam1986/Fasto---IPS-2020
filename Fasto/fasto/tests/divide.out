2
2
Interpreter error: Division with zero error: ; expected int and int, got int (2) and int (0) instead at line 5, column 21
